_data/simple-icons.json

```diff
@@ -4247,6 +4247,11 @@
             "hex": "29ABE2",
             "source": "https://www.webcomponents.org/"
         },
+        {
+            "title": "Webmin",
+            "hex": "7DA0D0",
+            "source": "https://github.com/webmin/webmin/tree/ed88a51ed13255d8dfe1143afe665fc261e992f7/images"
```

Comment:
I would personally prefer just the repository URL **or** this URL but just for the `master` branch (rather than the specific commit)